src/components/Nav.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { Route, NavLink, BrowserRouter } from "react-router-dom";
+import About from "./About";
+import App from "./app";
+
+
+const Nav = () => {
+  return (
+    <BrowserRouter>
+      <div>
+        <div className="navbar">
+<<<<<<< HEAD
+          <ul>
+            <li>
+              <NavLink to="/">Home</NavLink>
+            </li>
+            <li>
+              <NavLink to="/about">About</NavLink>
+            </li>
+          </ul>
+=======
+        <ul>
+          <li>
+            <NavLink to="/">Home</NavLink>
+          </li>
+          <li>
+            <NavLink to="/about">About</NavLink>
+          </li>
+        </ul>
+>>>>>>> a56d48e... eslint fixed, and navbar added css, change directory of app.css
```

Comment:
Can you resolve this conflict. It's the one causing problem when we run tests

Reply:
it has been resolved on bg-reactSetup-dep branch  except it hasn't been merged yet

Reply:
Ok no problem then. everything should work fine now